Exercise_A/index.html

```diff
@@ -1,16 +1,27 @@
 <!DOCTYPE html>
 <html class="no-js">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="" />
-  </head>
-  <body>
-    <!-- CODE HERE -->
 
-    <script src="" async defer></script>
-  </body>
-</html>
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <title></title>
+  <meta name="description" content="" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="style.css" />
+  <link rel="stylesheet" href="" />
+</head>
+
+<body>
+  <div class="a">
+    <a href="about.html">
+      <button>My Email</button>
+    </a>
+    <div class="outerbox">
+      hi
+    </div>
+  </div>
+
+  <script src="" async defer></script>
+</body>
+
+</html>
```

Comment:
Hey Alex 
Great work on this, you had an element which i was able to click and then move on to your about.html